Documentation/flexvolume.md

```diff
@@ -46,6 +46,23 @@ Continue with [configuring the FlexVolume path](#configuring-the-flexvolume-path
 Rancher provides an easy way to configure kubelet. The FlexVolume flag will be shown later on in the [configuring the FlexVolume path](#configuring-the-flexvolume-path).
 It can be provided to the kubelet configuration template at deployment time or by using the `up to date` feature if Kubernetes is already deployed.
 
+Rancher deploys kubelet as a docker container, you need to mount the host's flexvolume path into the kubelet image as a volume,
+this can be done in the extra_binds section of the kubelet cluster config.
+
+Configure the rancher deployed kubelet by updating the `cluster.yml` file kubetlet section:
+
+```yaml
+kubelet:
+ image: ""
+ extra_args:
+  volume-plugin-dir: /usr/libexec/kubernetes/kubelet-plugins/volume/exec
+  extra_binds:
+  - /usr/libexec/kubernetes/kubelet-plugins/volume/exec:/usr/libexec/kubernetes/kubelet-plugins/volume/exec
+```
+
+If you're using [rke](https://github.com/rancher/rke), run `rke up`, this will update and restart your kubernetes cluster system components, in this case the kubelet docker instance(s)
+will get restarted wit the new volume bind and volume plugin dir flag
+
 Continue with [configuring the FlexVolume path](#configuring-the-flexvolume-path) to configure Rook to use the FlexVolume path.
```

Comment:
I would like to see a sentence added here, as previously written, at least stating that the flexvolume dir for Rancher users is `/usr/libexec/kubernetes/kubelet-plugins/volume/exec` and that if this (Kubernetes) default path is used, no further configuration is needed on the operator and that if for whatever reason the user chooses a different path that it must be configured on the operator.

Reply:
agree, I've added in a section in the latest commit.

Reply:
see if its ok, otherwise I can rewrite it :) docs isn't my strong point.

Reply:
It's fine, no problem.